### DISCUSSION TITLE
「问题求助，求专业人员帮我解释一下这是什么情况」

### DISCUSSION THREAD

**Discussion by TOKLK** [share121/inter-knot#2003] [其他]
每次回帖或者发帖的时候总是显示这个，到底是什么原因啊？
![Screenshot_2024-08-14-05-36-39-750_mark via](https://github.com/user-attachments/assets/ce6c385f-14af-4c20-a682-350a9578bf30)

**Comment by Enigma-Soul** [comment]
人不行别怪路不平，没点魔法还来Github
